app/config/questions.py

```
@@ -0,0 +1,208 @@
+        "rubrica_key": "pregunta_20",
+        "field_name": "additional_info"
+    }
+}
+
+
+def get_question(question_number: int) -> Dict[str, Any]:
+    """Obtiene una pregunta específica por número"""
+    return WIZARD_QUESTIONS.get(question_number, {})
+
+
+def get_questions_by_type(question_type: str) -> Dict[int, Dict[str, Any]]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)